vpr/src/draw/draw_basic.cpp

```diff
@@ -1127,15 +1202,16 @@ bool is_flyline_valid_to_draw(int src_layer, int sink_layer) {
 }
 
 //Draws critical path shown as flylines.
-void draw_flyline_timing_edge(ezgl::point2d start, ezgl::point2d end, float incr_delay, ezgl::renderer* g) {
+void draw_flyline_timing_edge(ezgl::point2d start, ezgl::point2d end, float incr_delay, ezgl::renderer* g, bool skip_draw_delays) {
```

Comment:
It is better to mention the default value for the arguments that have one

bool skip_draw_delays/*=false*/

Reply:
fixed https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2526/commits/a608ba1a9b0be72e60900aa97374349b8b2e96e6